Fix PHP performance monitoring guides (#4637)
getsentry/sentry-docs: src/includes/performance/add-spans-example/php.mdx

## Add More Spans to the Transaction

The next example contains the implementation of the hypothetical `expensive_operation` function called from the code snippet in the previous section. Keep in mind that each individual span also needs to be manually finished; otherwise, spans will not show up in the transaction.

> You can choose the value of `op` and `description`.

Reviewer: ```suggestion
You can choose the values of `op` and `description`.
```

```php
function expensive_operation(): void
{
    $parent = \Sentry\SentrySdk::getCurrentHub()->getSpan();
    $span = null;

    // Check if we have a parent span (this is the case if we started a transaction earlier)
    if ($parent !== null) {
        $context = new \Sentry\Tracing\SpanContext();
        $context->setOp('some_operation');
        $context->setDescription('This is a description');
        $span = $parent->startChild($context);

        // Set the current span to the span we just started
        \Sentry\SentrySdk::getCurrentHub()->setSpan($span);
    }

    // Do the expensive operation...

    // We only have a span if we started a span earlier
    if ($span !== null) {
        $span->finish();

        // Restore the current span back to the parent span
        \Sentry\SentrySdk::getCurrentHub()->setSpan($parent);
    }
}
```

> This code checks for the existence of a parent span and if we have one, we create a new span and set it as the current span. We then call the expensive operation. Finally, we finish the span and restore the current span to the parent span.

Reviewer: Using first person (we) in that first sentence is a bit unclear since the sentence begins with the code being the thing taking action, so I've changed it to passive voice. 

```suggestion
This code checks for the existence of a parent span and if there is one, a new span is created and set as the current span. Then we call the expensive operation. Finally, we finish the span and restore the current span to the parent span.
```
